Frameworks/CoreGraphics/CGContextCairo.mm

```diff
@@ -1629,6 +1630,229 @@
     UNLOCK_CAIRO();
 }
 
+void CGContextCairo::_CGContextDrawStrokedPath(cairo_path_t* flatPath, int i, int prev, int total, CGFloat lineWidth) {
```

Comment:
This is an internal method.  It's called by CGContextReplacePathWithStrokedPath which uses cairo_copy_path_flat which always returns a valid pointer, but the result may have no data (data==NULL and num_data==0).  So I think we are good w/o a NULL check.